fail2ban/server/server.py

```diff
@@ -402,8 +434,14 @@ def getLogLevel(self):
 	# @param target the logging target
 	
 	def setLogTarget(self, target):
-		try:
-			self.__loggingLock.acquire()
+		with self.__loggingLock:
+			# don't set new handlers if already the same
+			# or if "INHERITED" (foreground worker of the test cases, to prevent stop logging):
+			if self.__logTarget == target:
+				return True
+			if target == "INHERITED":
```

Comment:
I don't think docs were adjusted to reflect having this new setting I am yet to comprehend